test/pageobject/SettingsScreen.ts

```diff
@@ -0,0 +1,57 @@
+export const GeneralSettingsScreen = {
+  setSessionTimeoutByIndex: async (index: number) => {
+    await browser.$("[class*='trigger@Select-module']").click();
+    const items = await browser.$$("[class*='item@Select-module']");
+    await items.at(index)?.click();
+  },
+};
+
+const Permission = (wrapped: WebdriverIO.Element) => ({
+  get detailsIcon() {
+    return wrapped.$("[class*='settings@list']");
+  },
+});
+
+export const PermissionControlSettingsScreen = {
+  get root() {
+    return browser.$("[class*='content@permissionsSettings']");
+  },
+
+  get permissionItems() {
```

Comment:
Would be cool to have `getPermissionByOrigin` method, to avoid using indices

Reply:
Done